website/search/views.py

```diff
@@ -245,7 +249,7 @@ def search_share_atom(**kwargs):
     query = build_query(q, size=RESULTS_PER_PAGE, start=start, sort=sort)
 
     try:
-        search_results = search.search_share(query, index='share_v1')
+        search_results = search.search_share(query, index=settings.SHARE_ELASTIC_INDEX)
```

Comment:
This also needs to stay share_v1 until the frontend is updated to deal with share_v2 data.